src/wapm.spec.ts

```diff
@@ -6,12 +6,13 @@
 import test from 'ava';
 import {existsSync} from 'fs';
 import {getClient} from './apollo';
-import {getLocation, getRegistryUrl, login, publish, setRegistryUrl} from './wapm';
+import {getLocation, getRegistryUrl, login, publish, setRegistryUrl, tokenLogin} from './wapm';
 import {loadEnv, loggedIn, logout, resolveFixture} from './test-utilities';
-
 //Environment variables
 const username = loadEnv('WAPM_USERNAME');
 const password = loadEnv('WAPM_PASSWORD');
+const userToken = loadEnv('WAPM_REGISTRY_TOKEN');
```

Comment:
If the token secret name in `.github/workflows/test.yml` is updated from `WAPM_REGISTRY_TOKEN`, make sure to update this too.